examples/tac_deploy/tac_run.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash -e
+
+minutes_to_start=$1
+participants=$2
+contract=$3
+registry=$4
+acn_node=$5
+identifier=$(python -c "import uuid; print(uuid.uuid4().hex)")
+tac_name=v1_$identifier
+PEER=""
+agents=""
+registry_flag="--local"
+
+help() {
+echo "
+Use tac_run.sh <minutes_to_start> <amount_of_participatns> [contract:noncontract,fetchai] [registry:local,remote] [acn_node:local,public]
+
+minutes_to_start - int
+amount_of_participatns - int, min 2
+contract - contract to use: noncontract,fetchai, ethereum. noncontract is default
+registry - local or remote, local is default
+acn_node - local or public, local is default
+"
+ exit 1;
+}
+
+function empty_lines {
+echo "
+
+"
+}
+
+print_options(){
+start_time=$(date)
+echo "Starting tac example.
+
+Start time: ${start_time}
+
+* minutes_to_start - ${minutes_to_start}
+* amount_of_participatns - ${participants}
+* contract - ${contract}
+* registry - ${registry}
+* acn_node - ${acn_node}
+* tac_name(autogenerated) - ${tac_name}
+
+
+"
+}
+
+check_options(){
+	if [ -z "$minutes_to_start" ]; then
+		echo "No minutes provided.";
+		help
+	fi
+	
+	if [ -z "$participants" ]; then
+		echo "No participants provided";
+	    help
+	fi
+	
+	case "$registry" in
+	  local)
+        registry_flag="--local"
+	    ;;
+	  remote)
+        registry_flag="--remote"
+	    ;;
+	  "")
+	  	registry=local
+	  	registry_flag="--local"
+	    ;;
+	  *)
+	  	echo "Incorrect registry!"
+	    help
+	    ;;
+	esac
+	
+	
+	case "$acn_node" in
+	  local)
+	     PEER=""
+	    ;;
+	  public)
+	     PEER="/dns4/acn.fetch.ai/tcp/9001/p2p/16Uiu2HAmVWnopQAqq4pniYLw44VRvYxBUoRHqjz1Hh2SoCyjbyRW"
+	    ;;
+	  "")
+	  	acn_node=local
+	  	PEER=""
+	    ;;
+	  *)
+	    echo "Incorrect acn node option!"
+	    help
+	    ;;
+	esac
+	
+	case "$contract" in
+	  fetchai)
```

Comment:
Add `ethereum` case similar to `fetchai`

Reply:
```
ethereum)
 	key_type=$mode
 	tac_controller_name="tac_controller_contract"
 	 tac_controller_skill_name="tac_control_contract"
 	 tac_participant_name="tac_participant_contract"
 ;;
```

Reply:
@5A11 
ethereum does not work for me, i used tac_contract_ethereum_local_registry_local_acn.sh as a reference and it does not work for me

Reply:
```
./examples/tac_deploy/tac_contract_ethereum_local_registry_local_acn.sh  2 2
```
```
Build completed!
Issuing certificate 'acn' for connection fetchai/p2p_libp2p:0.24.0...
Error: Cannot find private key with id 'fetchai'. Please use `aea generate-key fetchai` and `aea add-key fetchai` to add a private key with id 'fetchai'.
```

so we should add two keys: fetchai and ethereum?